package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa-ga",
-  "version": "2.0.3",
+  "version": "2.1.0-alpha",
```

Comment:
Any reason why not make a non alpha/beta release?

Reply:
I like to test it out, and check there are no bugs with this change then I can make a proper release